server/helpers/toyProblemsChecker.js

```diff
@@ -90,14 +90,30 @@ const checkToyProblems = async cohort => {
 
 const getReleasedToyProblems = async cohort => {
   const response = await githubQuery(
-    `https://api.github.com/repos/hackreactor/${cohort}-toy-problems/contents/`
+    `https://api.github.com/repos/hackreactor/${cohort}-toy-problems/commits?per_page=100`
   );
```

Comment:
Currently limits the query to return only last 100 commits. Need to paginate in a future iteration.